src/js/modules/infragistics.ui.editors.js

```diff
@@ -6071,15 +6068,15 @@
 
 					//Notify
 					this._sendNotification("warning",
-						$.ig.util.stringFormat($.ig.Editor.locale.maxValExceedSetErrMsg,
+						$.ig.util.stringFormat(this._getLocaleValue("maxValExceedSetErrMsg"),
 							this.options.maxValue));
 				} else if (!isNaN(this.options.minValue) &&
 					newValue / this.options.displayFactor < this.options.minValue) {
 					newValue = this.options.minValue * this.options.displayFactor;
 
 					//Notify
 					this._sendNotification("warning",
-						$.ig.util.stringFormat($.ig.Editor.locale.minValExceedSetErrMsg,
+						$.ig.util.stringFormat(this._getLocaleValue("maxValExceedSetErrMsg"),
```

Comment:
Please, check if you didn't change minValExceedSetErrMsg with maxValExceedSetErrMsg